UI/index.erb

```diff
@@ -210,13 +210,19 @@
 		<script src="lib/jsutils/underscore-min.js">
         </script>
 
-        <script src="js/utils.js">
+        <script src="js/utils/utils.js">
         </script>
         <script src="lib/angular/angular.min.js">
         </script>
-        <script src="js/controllers.js">
+        <script src="js/app/app.js">
         </script>
-        <script src="js/filters.js">
+
+        <script src="js/controllers/controllers.js">
+        </script>
+        <script src="js/factories/factories.js">
+        </script>
```

Comment:
I see that you haven't added the `type` attribute saying that this is a javascript file. is that redundant info?